apps/api/src/app/workflows-v2/usecases/validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase.ts

```diff
@@ -123,6 +125,9 @@ export class PrepareAndValidateContentUsecase {
       controlValueToValidPlaceholders,
       controlValueToContentIssues
     );
+
+    this.overloadMinValueIssues(jsonSchemaDto, flatSanitizedControlValues, controlValueToContentIssues);
```

Comment:
The thing that I don't really like, but this part of the code validates the control values with the schema min values and returns the issues.